subsys/bluetooth/services/ras/rrsp/ras_rd_buffer.c

```diff
@@ -194,8 +194,9 @@ static void subevent_data_available(struct bt_conn *conn,
 				    struct bt_conn_le_cs_subevent_result *result)
 {
 	LOG_DBG("Procedure %u", result->header.procedure_counter);
-	struct ras_rd_buffer *buf =
-		rd_buffer_get(conn, result->header.procedure_counter, false, true);
+	uint16_t ranging_counter =
+		bt_ras_rreq_get_ranging_counter(result->header.procedure_counter);
```

Comment:
`bt_ras_rreq_get_ranging_counter` is now a bit of an unfortunate name, since there's nothing RREQ-specific about it.
Probably not worth breaking API over this.